packages/cli-platform-android/src/commands/buildAndroid/startServerInNewWindow.ts

```diff
@@ -16,34 +17,50 @@ export function startServerInNewWindow(
     ? 'launchPackager.bat'
     : 'launchPackager.command';
   const packagerEnvFilename = isWindows ? '.packager.bat' : '.packager.env';
-  const portExportContent = isWindows
-    ? `set RCT_METRO_PORT=${port}`
-    : `export RCT_METRO_PORT=${port}`;
+  const packagerEnvFileExportContent = isWindows
+    ? `set RCT_METRO_PORT=${port}\nset PROJECT_ROOT=${projectRoot}\nset REACT_NATIVE_PATH=${reactNativePath}`
+    : `export RCT_METRO_PORT=${port}\nexport PROJECT_ROOT=${projectRoot}\nexport REACT_NATIVE_PATH=${reactNativePath}`;
+  const nodeModulesPath = path.join(projectRoot, 'node_modules/.bin');
```

Comment:
Nit: Use `path.join` to join the last part as well?
```suggestion
  const nodeModulesPath = path.join(projectRoot, 'node_modules', '.bin');
```